src/wp-content/themes/twentyeleven/showcase.php

```diff
@@ -16,7 +16,16 @@
  */
 
 // Enqueue showcase script for the slider.
-wp_enqueue_script( 'twentyeleven-showcase', get_template_directory_uri() . '/js/showcase.js', array( 'jquery' ), '20211130' );
+wp_enqueue_script(
+	'twentyeleven-showcase',
+	get_template_directory_uri() . '/js/showcase.js',
+	array( 'jquery' ),
+	'20211130',
+	array(
+		'in_footer' => false, // Because involves header.
+		'strategy'  => 'defer',
+	)
```

Comment:
Oh, this actually was originally added to the `head`, so in WP<6.3 this will actually get printed in the footer since the `$args` array is truthy for `$in_footer`. Nevertheless, the function's contents are wrapped in `jQuery.ready()` so it makes no difference when it loads.